ExeterTeam25/group-software-project
# Game masters

Added functionality so only users in the group 'game_master' and 'developer' can access the challenge page. Regular users should not be able to see the button or be able to access through the URL

To test:

1. Ensure you're logged in as a superuser
2. On the admin page, add the following groups
3. 'user', 'game_master', 'developer'
4. Under your profile, ensure you are in all of the groups, or at least in game_master
5.  You should be able to see the add challenge button and access the page
6. Create another profile or change your own profile so that it is only a user and doesn't have these permissions
7. You should not be able to see the link or access through the URL
